cmd/rootMain.go

```diff
@@ -125,6 +127,9 @@ func rootMain(cmd *cobra.Command, args []string) exitCode {
 	// Configure logrus to emit simple text
 	log.SetFormatter(&log.TextFormatter{DisableTimestamp: true})
 
+	// Ignore tty errors so we aren't suspended when prompting for input from the background.
+	signal.Ignore(syscall.SIGTTIN, syscall.SIGTTOU, syscall.SIGTSTP)
```

Comment:
Not sure I understand this. When would those TTY errors happen? What do those errors do? A longer comment or a reference would help.

Reply:
Thanks, this helps. So what is `SIGTTOU` for?

Reply:
Updated it. The actual behavior's weird, because we're getting SIGTTOU but everything I read suggests I should be getting SIGTTIN. I think it makes sense to ignore both (but I don't think we need to ignore SIGTSTP).